src/engine.cc

```diff
@@ -77,7 +87,13 @@ void EngineController::PopulateOptions(OptionsParser* options) {
   defaults->Set<int>(Search::kAllowedNodeCollisionsStr, 32);  // Node collisions
 }
 
-SearchLimits EngineController::PopulateSearchLimits(int /*ply*/, bool is_black,
+double move_weight(int ply, float peak, float left_width, float right_width) {
```

Comment:
`ComputeMoveWeight` instead of `move_weight`.
Also it's better to move that function into anonymous namespace.
Also it makes sense for it to return float rather than double.